lib/lobbyist/v2/skipped_payment.rb

```diff
@@ -0,0 +1,18 @@
+module Lobbyist
+  module V2
+
+    class SkippedPayment < Lobbyist::V2::Base
+      attr_accessor :id, :company_id, :admin_user_id, :reason, :source, :transaction_status
+      attr_accessor :status, :amount, :transaction_date, :created_at, :updated_at
+
+      def self.list(company_id, params = {})
+        create_collection_from_response(get("/v2/companies/#{company_id}/company_transactions/skip-payments.json", params))
+      end
+
+      def self.find(company_id, params = {})
+        create_collection_from_response(get("/v2/companies/#{company_id}/company_transactions/active-skip-payment.json", params))
```

Comment:
for find we do not use 'create_collection_from_response'
`create_from_response(get(/v2/companies/#{company_id}/company_transactions/active-skip-payment.json", params))`
and plz make necessary changes on the accounts in parsing the response

Reply:
ok it's done. So merging it.